templates/info.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Linley/Pitts Wedding Info</title>
+    <link rel="stylesheet" href="styles/main.css" type="text/css"/>
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lato|Lora" rel="stylesheet">
+  </head>
+  <body>
+    <section class="sitemap-header">
+      <span class="sitemap-name">David Linley</span>
+      <div class="link-container">
+        <a href="/" class="sitemap-link">RSVP</a>
+        <a href="#" class="sitemap-link">Registry</a>
+        <a href="#" class="sitemap-link">Info</a>
+      </div>
+      <span class="sitemap-name">Michelle Pitts</span>
+    </section>
+
+    <div class="banner-image-container">
+      <div class="banner-image">
+
+      </div>
+    </div>
+    <div class="card">
+      <div class="info">
+        <div class="card-header">
+          <h2>Just the Basics</h2>
+        </div>
+        <p><em>Who: </em>David Linley and Michelle Pitts</p>
+        <p> <em>What: </em>A wedding, followed by a buffet dinner, an open bar, and lots of dancing and merriment </p>
+        <p><em>When: </em>The ceremony will take place at 4:00 pm on Saturday, August 11, 2018</p>
+        <p> <em>Where: </em>18 Mill Street, Angus, Ontario </p>
+      </div>
+    </div>
+    <div class="card">
+        <div class="card-header">
+          <h2>Accommodations</h2>
+        </div>
+
+        <div id="map" style="height: 400px; width: 100%;"></div>
+
+        <p>
+          Several hotels are within easy driving distance of our pictuersque venue. We recommend the Knights Inn in downtown Angus, but the Angus Inn is another excellent option. Both are located a mere three-minute drive
+          from the venue. If you'd prefer something a little more cosmopolitan, several major hotel chains are present in Barrie, approximately 20 minutes away. Visitors looking for more homey accomodations might enjoy
+          one of the many bed and breakfasts in nearby historic Alliston. Wherever you choose to stay, a driver will be available throughout the evening to ensure you arrive safely at your destination. </p>
+        <p>Whatever lodgings you'd prefer, we recommend booking as soon as possible. Between the military posting season and the smaller size of these locations, rooms book up quickly in the summer months.</p>
```

Comment:
Should probably indent the paragraph text, separating it from the <p> tag for consistency.